docs/source/examples.orca_cli.md

```diff
@@ -1,136 +0,0 @@
-# ORCA via command line 
-
-## 1. Evaluate atomic configurations sequentially on a single core
-
-The following command will evaluate `configs.xyz` with the ORCA DFT code. The Workflow code called with `wfl ref-method orca-eval ... ` is an extension of the [ASE ORCA calculator](https://wiki.fysik.dtu.dk/ase/ase/calculators/orca.html#module-ase.calculators.orca). The extended calculator automates file handling: each structure is evaluated in a separate directory and only specified files are kept. 
-
-```
-scratchdir=/scratch-ssd/my_dir_on_scratch
-mkdir -p $scratchdir
-
-wfl -v ref-method orca-eval \
-    --scratchdir $scratchdir \
-    --workdir orca_workdir \
-    --output-prefix dft_ \
-    --orcasimpleinput "UKS B3LYP def2-SV(P) def2/J D3BJ" \
-    --orcablocks "%scf Convergence Tight SmearTemp 5000 end" \
-    --calculator-exec "path/to/orca/orca_5.0.0/orca" \
-    --output-file configs.dft.xyz \
-    --keep-files default \
-    configs.xyz 
-```
-
-The parameters are 
-
-* `workdir` - this directory is where directories for each separate DFT calculation will be based. 
-* `orcasimpleinput` - [ORCA simple input](https://sites.google.com/site/orcainputlibrary/general-input?authuser=0#h.rafay5vzyzkw) line, during the calculation automatically prepended with `!`.
-* `orcablocks` - [ORCA block input](https://sites.google.com/site/orcainputlibrary/general-input?authuser=0#h.k0jcjw8bcgcx), the rest of ORCA input file, excluding the coordinate definition.
-* `output_prefix` - prefix to the results' keys in `atoms.info` and `atoms.arrays`. Here DFT results will be saved to `dft_energy`, `dft_forces` and `dft_dipole`. 
-* `calculator-exec` - path to ORCA executable.
-* `scratchdir` - directory on a SSD for more efficient calculations. 
-* `input` - `.xyz` file to be evaluated. 
-* `output` - `.xyz` to save configs with results to. This command will refuse to overwrite if the file exists. 
-* `--keep-files default` - only `.inp`, `.out`, `.ase`, `.engrad`, `.xyz` and `_trj.xyz` files will be saved, where applicable, upon success. All files will be kept if the calculation failed.
-* `wfl -v ref-method orca-eval` - Workflow command to evaluate DFT, in verbose mode. 
-
-With this command each structure will be evaluated sequentially on a single core. 
```

Comment:
Are you setting `num_python_subprocesses=1` explicitly, or are we relying on `WFL_NUM_PYTHON_SUBPROCESSES` not to be set?